src/main/java/io/gravitee/common/util/EnvironmentUtils.java

```diff
@@ -91,27 +96,20 @@ public static Map<String, Object> getAllProperties(PropertySource<?> aPropSource
     public static boolean hasMatchingTags(Optional<List<String>> configuredTags, Set<String> tags) {
         if (configuredTags.isPresent()) {
             List<String> tagList = configuredTags.get();
-            if (tags != null && !tags.isEmpty()) {
-                final List<String> inclusionTags = tagList.stream().map(String::trim).filter(tag -> !tag.startsWith("!")).toList();
 
-                final List<String> exclusionTags = tagList
-                    .stream()
-                    .map(String::trim)
-                    .filter(tag -> tag.startsWith("!"))
-                    .map(tag -> tag.substring(1))
-                    .toList();
-
-                if (inclusionTags.stream().anyMatch(exclusionTags::contains)) {
```

Comment:
Do we really want to remove this check ?

Reply:
Yes, we do. The inclusionTags and exclusionTags extracted from a list of tags set in a gravitee.yaml. It is not an error due to the API tags and even so, I don't see the purpose of raising an exception for such use case while actually you could just return false in this case.